applications/MappingApplication/tests/cpp_tests/test_nearest_element_aux_classes.cpp

```diff
@@ -103,50 +176,112 @@ KRATOS_TEST_CASE_IN_SUITE(NearestElementInterfaceInfo_ValidProjectionExists, Kra
     KRATOS_CHECK_DOUBLE_EQUAL(sf_values[2], 0.4);
 }
 
-KRATOS_TEST_CASE_IN_SUITE(NearestElementInterfaceInfo_Approximation, KratosMappingApplicationSerialTestSuite)
-{
-    const double dist = 1.33;
-    Point coords(0.0, -dist, 0.0);
+// KRATOS_TEST_CASE_IN_SUITE(NearestElementInterfaceInfo_Approximation, KratosMappingApplicationSerialTestSuite)
```

Comment:
this test does not compile with the current state, it requires some more implementations to work, therefore I would leave it to a future PR to reenable it